src/test/java/com/stripe/functional/AccountTest.java

```diff
@@ -1,7 +1,11 @@
 package com.stripe.functional;
 
-import com.stripe.BaseStripeFunctionalTest;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+import com.stripe.BaseStripeMockTest;
 import com.stripe.exception.StripeException;
+import com.stripe.net.APIResource;
 import com.stripe.model.*;
 import junit.framework.Assert;
 import org.junit.Test;
```

Comment:
Kind of a nit, but at some point in the future I'd like to enforce Google's Java style in the project. Can you reorder the import like in the other tests?

1. All static imports first
2. All imports in aphabetical order (`com.` / `org.` prefixes are ignored)
3. Separate import "groups" with a new line (in most cases the groups will be java, junit, then stripe)
4. Don't use `.*`, explicitly import all the classes you need.